package.json

```diff
@@ -10,7 +10,7 @@
     "test": "truffle test",
     "test-stablex": "truffle test test/stablex/*",
     "test-snapp": "truffle test test/snapp/*",
-    "coverage": "solidity-coverage",
+    "coverage": "truffle run coverage --network development --temp build",
```

Comment:
```
--temp build
```
means, "write build artifacts to the `build` folder." In SC 0.7.0 these are normally written to a temporary folder called '.coverage_artifacts' and thrown away. 

This flag needs to be set because your migrations rely on artifacts being available via the `build` path here:
https://github.com/gnosis/dex-contracts/blob/5ab2c4e231c96621fad6fada12100de2b97fdda6/src/index.js#L9-L13

NB: The `build` folder **gets deleted** after the coverage command is run (relevant if you're sometimes storing real network deployment data there).

Reply:
I wonder if it would be better to build the artefacts explicitly (so that they persist) and having coverage use the throw_away folder. Otherwise now we would delete artefacts, even if we `truffle compiled` them before, right?

Reply:
@fleupold 👍 

In c77baee531755c31304d3e48acdaeb93cea90f61 have removed the `--temp build` flag after modifying a `require` path in test/utilities to load encoding.js directly from its file instead of from `src/index`.  

That looks like the only place Node wasn't able to resolve paths correctly when using an alternative location for artifacts.

The build folder won't be touched or deleted by coverage anymore.